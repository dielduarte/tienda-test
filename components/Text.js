import styled from 'styled-components';

export const Regular = styled.h3`
  font-size: 22px;
  font-weight: 100;
`;

export const Small = styled.h3`
  font-size: 16px;
  font-weight: 100;
`;
