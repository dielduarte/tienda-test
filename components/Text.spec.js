import { shallow } from 'enzyme';
import 'jest-styled-components';

import { Regular, Small } from './Text';

describe('Text', () => {
  it('Regular', () => {
    const wrapper = shallow(<Regular />);
    expect(wrapper).toMatchSnapshot();
  });

  it('Small', () => {
    const wrapper = shallow(<Small />);
    expect(wrapper).toMatchSnapshot();
  });
});
