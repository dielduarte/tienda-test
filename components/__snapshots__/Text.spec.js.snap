// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Regular 1`] = `
.c0 {
  font-size: 22px;
  font-weight: 100;
}

<h3
  className="c0"
/>
`;

exports[`Text Small 1`] = `
.c0 {
  font-size: 16px;
  font-weight: 100;
}

<h3
  className="c0"
/>
`;
